src/main/java/me/botsko/prism/Prism.java

```diff
@@ -264,8 +279,12 @@ public static void log(String message) {
      */
     public static void warn(String message) {
         log.warning("[" + getPrismName() + "] " + message);
+        prismLog.warning(message);
+    }
+    public static void warn(String message, Exception e) {
```

Comment:
Codacy found an issue: ['METHOD_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)